packages/lib-feature-flags/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@karhoo/lib-feature-flags",
+  "version": "0.10.0",
+  "description": "Feature flags library for Karhoo web apps",
```

Comment:
Can we make nicer description with emoji and would be nice if you fill in keywords

Reply:
@juliamaksimchik Heh, ok I'll add emojis. :)
But I'm not sure about what exactly should be in keywords section?